sources/niUI/src/WidgetListBox.cpp

```diff
@@ -449,7 +453,8 @@ tU32 cWidgetListBox::GetSelection(tU32 anIndex) const
 ///////////////////////////////////////////////
 tBool cWidgetListBox::SetSelected(tU32 anSelection)
 {
-  ClearSelection();
+  // here we don't need to call the NotifySelChange() method, cause the AddSelection will do it.
+  mvSelection.clear();
```

Comment:
Seems sensible, but I think it'd be clearer if `ClearSelection` becomes `ClearSelection(tBool abNotifySelChange)`.